Solutions/Endjin.SpecFlow.Selenium.Framework.Net45/Navigation/NavigationMap.cs

```diff
@@ -31,6 +31,8 @@ private PageMapEntry HomePage
             get;
         }
 
+        public virtual string SpaRoutingMarker => null;
+
```

Comment:
Shouldn't SpaRoutingMarker be Protected - it's only ever going to be set by the inheriting class?